### PR TITLE
Update login note

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

It would look like this

Screen Shot 2022-03-23 at 9 35 27

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: web/src/components/login-page/index.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ export const LoginPage: FC = memo(function LoginPage() {
               />
               {isPlayEnvironment && (
                 <div className={classes.note}>
-                  Try with <strong>play</strong> name if you want to join the
+                  Try project name <strong>play</strong> if you want to join the
```
Comment (knanao) [inline web/src/components/login-page/index.tsx:104]: nits
```suggestion
                  Try to input <strong>play</strong> as project name if you want to join the
```

**Review by knanao — COMMENTED** [review]
Is better off `textAlign: "left"` ?
https://github.com/pipe-cd/pipecd/pull/3424/files#diff-859b4e56fe1e23be4f03551006e60f9de7a224b2d91542372eefb519fb58cdbbR43

File: web/src/components/login-page/index.tsx (hunk 1 above)
Comment (khanhtc1202) [reply web/src/components/login-page/index.tsx:104]: Nice idea, but this text is a bit too long to be shown in a single line 🙈

Comment (knanao) [reply web/src/components/login-page/index.tsx:104]: Oh, I got it.
Does it not also work if you set `testAlign: "left"`?

Comment (khanhtc1202) [reply web/src/components/login-page/index.tsx:104]: align right
Screen Shot 2022-03-23 at 9 32 10

align left
Screen Shot 2022-03-23 at 9 32 53

🙈

Comment (knanao) [reply web/src/components/login-page/index.tsx:104]: 😇 
allright, let's keep it that way.

Comment (nghialv) [inline web/src/components/login-page/index.tsx:104]: or just 😄 

```
Enter <strong>play</strong> if you want to join the playground environment
```

Comment (khanhtc1202) [reply web/src/components/login-page/index.tsx:104]: `Enter` is a bit odd to me, so lets me keep it as is 😄

Comment (khanhtc1202) [reply web/src/components/login-page/index.tsx:104]: But, maybe `Input <strong>play</strong> if you want to join the playground environment` is good. How do you think? @nghialv @knanao

Comment (knanao) [reply web/src/components/login-page/index.tsx:104]: Looks good!

Comment (khanhtc1202) [reply web/src/components/login-page/index.tsx]: addressed by https://github.com/pipe-cd/pipecd/pull/3424/commits/2e02521956bcdbe44baf69112c9fcc1d952969d0 🙌

**Comment by nghialv** [comment]
Nice.

/approve

**Pull request merged**